### ISSUE TITLE
Screenreader does not read title of fields of contact and participation form

### ISSUE THREAD

**Issue by hmkay** [liqd/digitalstrategie#260]
**URL:** https://digitalstrategie-dev.liqd.net/kontakt/
**user:** participant with screenreader
**expected behaviour:** screenreader reads fields of fields in order for people using screen reader to know where to put in their name
**behaviour:** the headline of fields are not being read out - 
**important screensize:**desktop
**device & browser:** macOS Monterey, using VoiceOver as Screenreader
**Comment/Question:** I am not very well trained in this so I am unsure if I got it right...or how are people able to know what kind of field they are in?

Screenshot?

**Comment by phillimorland** [comment]
@hmkay you got it right, it doens't work
